lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -188,6 +188,9 @@ void sendAttributeChanged(
         notifyWithoutExceptions<Ufe::Attributes>(
             Ufe::AttributeConnectionChanged(ufePath, changedToken.GetString()));
     } break;
+    default: {
```

Comment:
The only enumerant not covered is kMetadataChanged.  I think it would be better for maintainability to remove the catch-all default and explicitly add kMetadataChanged.